EnglishWebRoot/js/viewSubject.js

```diff
@@ -160,10 +164,20 @@ document.addEventListener("deviceready", function () {
             classes += " unlocked";
         }
 
+        for(var iCount = 0; iCount < levelNameEnglish.length; iCount++){

```

Comment:
Add some comment, mentioning that we have assumed English to be base way to structure the content and that we are overriding language to be specific

Reply:
Added comments